packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, Fragment, useRef } from 'react';
+import { useHover } from 'usehooks-ts';
+import { qtMerge, Text, useBreakpoint } from '@deriv/quill-design';
+import { Clipboard, Tooltip } from '../../../../../components/Base';
+import EditIcon from '../../../../../public/images/ic-edit.svg';
+
+type TMT5TradeDetailsItemProps = {
+    className?: string;
+    label?: string;
+    value: string;
+    variant?: 'clipboard' | 'info' | 'password';
+};
+
+const MT5TradeDetailsItem: FC<TMT5TradeDetailsItemProps> = ({ className, label, value, variant = 'clipboard' }) => {
+    const { isDesktop } = useBreakpoint();
+    const hoverRef = useRef(null);
+    const isHovered = useHover(hoverRef);
+    return (
+        <div
+            className={qtMerge(
+                'flex items-center h-1600 justify-between bg-system-light-secondary-background p-[5px] pl-400',
+                className
+            )}
+        >
+            {variant !== 'info' && (
+                <Fragment>
+                    <Text colorStyle='subtle' size='sm'>
+                        {label}
```

Comment:
This is optional as well so we need to check for the null situation.
e.g. label || ""